plugins/elementspath/plugin.js

```diff
@@ -222,7 +222,7 @@
 			}
 
 			var space = getSpaceElement();
-			space.setHtml( html.join( '' ) + emptyHtml );
+			space.setHtml( CKEDITOR.tools.htmlSafeByReview(html.join( '' ) + emptyHtml, 'The Html elements are all created from template using safe internal values') );
```

Comment:
There's a template output so let's legacy convert it.

Reply:
Legacy converted